tests/datasets/test_errors_processor.py

```diff
@@ -311,6 +312,7 @@ def build_result(self, meta: KafkaMessageMetadata) -> Mapping[str, Any]:
                 "geo.country_code",
                 "geo.region",
                 "geo.city",
+                "geo.subdivision",
```

Comment:
you added this field to the processor test but are there tests querying for them?